src/commands/script.rs

```diff
@@ -113,7 +121,7 @@ impl Script {
             CommandType::Execute => {
                 let command_string = if self.should_source {
                     // when sourcing, just return the full body.
-                    let mut command = vec![String::from("."), self.path.clone()];
+                    let mut command = vec![String::from("source"), self.path.clone()];
```

Comment:
For reading comprehension, `source` should be anbalagous to `.` but more self explanatory.

Reply:
`source` is non-posix, so we should stick with dot. See [the posix spec](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_18_01).

Reply:
of course `zsh` and `fish` are not completely posix compliant, but I think we should strive to use posix semantics unless we have a strong reason not to.